### PR TITLE
Signatures examples csharp

### PR DESCRIPTION
**Pull request by mickhansen**

### CODE REVIEW

**Comment by sgryt** [comment]
> @sgryt I'm using `cd src/examples && dotnet build` to validate that the C# examples are correct, but running into an issue with `error CS8802: Only one compilation unit can have top-level statements`. Do you have any suggestions?

Have you tried putting the example code in namespaces ?

File: src/examples/createSignatureOrder/basic.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+using Criipto.Signatures;
```
Comment (sgryt) [inline src/examples/createSignatureOrder/basic.cs:1]: Maybe a sample-specific namespace here (same comment for all .cs files) will fix the compilation issue.

Comment (mickhansen) [reply src/examples/createSignatureOrder/basic.cs:1]: @sgryt That won't work, the file has top level code which isn't allowed with a namespace

Comment (sgryt) [reply src/examples/createSignatureOrder/basic.cs:1]: I'm confused now: Aren't we trying to get rid of top-level code (to fix the compilation error) ??

**Comment by mickhansen** [comment]
@sgryt I will add the rest of the examples in another PR

**Pull request merged**